8/alpine/Dockerfile

```
@@ -23,7 +23,7 @@
 # FIXME(oleg_nenashev): This is not an official AdoptOpenJDK Docker Image.
 # There is no official Alpine images at the moment.
 # Needs upgrade when/if there is an official alpine image.
-FROM adoptopenjdk/openjdk8:x86_64-alpine-jdk8u322-b06
+FROM adoptopenjdk/openjdk8:x86_64-alpine-jdk8u332-b09
```

```suggestion
FROM eclipse-temurin:8u332-b09-jre-alpine
```

There is (now) an official temurin alpine distribution (only Intel though, but @gounthar is on the case for ARM support in the future ;) )

---

I've not been able to make the tests pass with the change from the adoptopenjdk to the eclipse-temurin image.  I had tried it previously without success and decided that it would be better to leave it for another pull request.

---

No problem. Then let's proceed to merging this PR: I've opened https://github.com/jenkinsci/docker-ssh-agent/pull/119 as a draft for the switch to temurin then